pkg/machines/libvirt.es6

```diff
@@ -332,6 +324,53 @@ LIBVIRT_PROVIDER = {
         };
     },
 
+    SET_VCPU_SETTINGS ({ name, connectionName, count, max, sockets, cores, threads, isRunning }) {
```

Comment:
Omitting override in OVIRT_PROVIDER means, that this will be used for ovirt-VMs as well. And it will not work.

Reply:
Seems to be implemented for oVirt now.